iprado78/fullstack-iex-stock-ticker
# feat: tabs and historical by day

## Description


* Creates the Tabs component and the HistoricalByDay component.
* `<Tabs>` expects multiple React components as children and a corresponding array of labels for the table button
* `<HBD>` functions very similarly to SummaryStats -- it has a corresponding api that expects the ticker symbol and the a date (string). It makes use of `DatePicker` (a external react component) and `dayjs` (a date library similar to momentjs, but smaller) 
* The dates are limited to only monday-friday, and dates that are in the past, because the api is looking up stock price data, which isn't available on weekends or in the future 
* I haven't re-added the styles because they're on Ignacio's local... and it didn't seem like necessary learning opportunity   
* still need to improve the styling of the calendar input...

👉  You will need to run npm install once this branch is merged

## Related Issues




- Closes #13 
- Closes #14 

## Testing



1. Open build
2. Search & select a stock ticker
3. Click between summary and historical and see them switch 
4. Change the day in the calendar & see the data update
5. Change the stock ticker & see the data update
